### DISCUSSION TITLE
Somes images missing from Export as markdown or html

### DISCUSSION THREAD

**Discussion by techepet** [outline/outline#7384] [Self Hosting]
On my self hosted site, I can see all page and images perfectly, however when I export as either markdown or html, both the whole site or individial pages, some pages, some of the images are missing and just showing the link as broken, but when i go into the folder containing the images on the export, the images are there. If feels like the problem is it creates too long of paths on windows and the way it treats all images as the same name,

Is there any way to change on the site how all images are saved or paths so that exports will work correctly?

atached an image for reference

![image](https://github.com/user-attachments/assets/ae5b09c4-1e79-49fb-9e34-6478752a6b9d)

**Comment by tommoor** [comment]
What are you using to view the markdown here? It feels like the problem is in this software not correctly decoding spaces in the image path

**Reply by techepet** [reply]
I tried it on Typora, or notion, or confluence. Same problem.
Perhaps this image better ilustrates the difference on the path, and how one image seems to be missing the extension.

![image](https://github.com/user-attachments/assets/ed92effe-e695-4e40-b286-410919471022)

**Reply by tommoor** [reply]
Maybe encoding the URL isn't the right thing to do then when the files are on disk, will check into it

**Reply by techepet** [reply]
Actually, I think I found the problem, so each of the images failing had an image caption, and on the export it seemed to be pointing to the name in the caption. The moment I removed the caption and did an export again, all images were rendered ok. 

The only things that don't get translated now are the info, tip and warning blobs.
![image](https://github.com/user-attachments/assets/4a99e6bc-5522-41fd-b38a-d7599fe75c22)
